scripts/!mods_preload/msu_mod_skill_container.nut

```diff
@@ -42,38 +42,7 @@ gt.MSU.modSkillContainer <- function ()
 	::mods_hookNewObject("skills/skill_container", function(o) {
 		o.m.BeforeSkillExecutedTile <- null;
 		o.m.IsExecutingMoveSkill <- false;
-
-		o.m.EventsDirectory <- {
```

Comment:
Perhaps we could record this list somewhere because it's useful to see all the vanilla events at a glance

Reply:
Done